src/components/ui/Uploader.js

```diff
@@ -28,44 +30,99 @@ const Description = styled.div`
     color: #FFFFFF;
 `;
 
+const ImageWrapper = styled.div`
+    position: relative;
+    width: 100%;
+    height: 100%;
+`;
+
+const TargetImage = styled.img`
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+`;
+
+const OverlayCanvas = styled.canvas`
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+`;
+
 class Uploader extends React.Component {
     constructor(props) {
         super(props);
 
         this.state = {
+            imgUrl: null,
         };
     }
 
+    async updateImage(imgUrl) {
+        this.props.updateStatus(Status.LOADING_IMAGE);
+        this.setState({ imgUrl: imgUrl });
+        const image = await faceApi.fetchImage(imgUrl);
+        const wrapper = document.getElementById('imageWrapper');
+        const canvas = document.getElementById('overlay' + this.props.index);
+        const displaySize = {
+            width: wrapper.clientWidth,
+            height: wrapper.clientHeight,
+        };
+        console.log(JSON.stringify(displaySize));
```

Comment:
please remove it if it is a debugging purpose

Reply:
Thanks! Done.